core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemActor.java

```diff
@@ -609,4 +599,12 @@ public int getNumberOfBackups() {
 		return numberOfBackups;
 	}
 
+	@IbisDoc({"9", "Can be set to 'encode' or 'decode' for actions "+ACTION_READ1+", "+ACTION_WRITE1+" and "+ACTION_APPEND+". When set the stream is base64 encoded or decoded, respectively", ""})
+	public void setBase64(String base64) {
+		this.base64 = base64;
+	}
+	public String getBase64() {
+		return base64;
+	}
```

Comment:
Ik snap dat je er nu niets mee wilt doen maar hier is de base64 pipe toch voor, dit moet weg!